demo-app/frontend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14 as build
```

Comment:
Node 14 is in maintenance mode, I'd recommend we upgrade this to either 18 or 20 https://nodejs.dev/en/about/releases/